tools/sql_extraction/src/main/kotlin/DataFlowEngine.kt

```diff
@@ -20,6 +24,42 @@ class DataFlowEngine {
             .map { QueryUsages(it.key, it.value.sorted()) }
     }
 
+    /**
+     * Visits method scope from an Antlr visitor
+     *
+     * @param[visitChildren] function to visit the rest of the method
+     */
+    fun visitMethod(visitChildren: () -> Unit) {
+        visitScope(visitChildren)
+    }
+
+    /**
+     * Associates [paramName] as a method argument for the most recently called [visitMethod]
+     */
+    fun addMethodParameter(paramName: String) {
+        LOGGER.debug("add method parameter $paramName")
```

Comment:
nit: This comment is intentionally leave there for debugging purpose? 
If just for logging, should we use LOGGER.log (if there is a log method :)).. and we should use "Add method parameter $paramName."